### DISCUSSION TITLE
Flyout not coming up

### DISCUSSION THREAD

**Discussion by drei97** [AvaloniaUI/Avalonia#13821] [Q&A] [answered]
Hi,

I have recently implemented my first Flyout for a TextBox element within a DataGridTemplateColumn

```
<DataGridTemplateColumn>
    <DataGridTemplateColumn.CellTemplate>
        <DataTemplate>
            <TextBlock Text="{Binding Something">
                <TextBlock.ContextFlyout>
                    <Flyout>
                        <views:MyFirstFlyout />
                    </Flyout>
                </TextBlock.ContextFlyout>
            </TextBlock>
        </DataTemplate>
    </DataGridTemplateColumn.CellTemplate>
</DataGridTemplateColumn>
```

The issue is that the Flyout only comes up if TextBlock contains some text. If the Text-property of TextBlock is empty (null), the Flyout keeps hidden.

What and how to change that the Flyout also pops up when TextBlock.Text is empty?

3.97

**Comment by timunie** [comment, marked as answer]
you need a Background to make the hittest work. Transparent is okay, but 'null' will not trigger any hittest
